src/Web/Views/Shared/_FraudProtectionIO.cshtml

```diff
@@ -35,6 +35,12 @@
                         </div>
                     </section>
                 </div>
+                @if (@io.CallTime > 0)
+                {
+                    <div class="row">
+                        <h5>The above call took @io.CallTime ms</h5>
```

Comment:
```suggestion
                        <h5>The response time for the above request is @io.CallTime ms</h5>
```

Reply:
changed this to be API call duration based on Chad's comment below